src/redux/accessoriesRedux.tsx

```diff
@@ -36,11 +36,17 @@ export const updateAmountAccessoryRequest = (accessory) => {
   };
 };
 
+//action types
+
+export interface LoadAccessoriesAction {
+  type: "LOAD_ACCESSORIES"
+  payload: CartItem[]
+}
+
 // reducer
-const accessoriesReducer = (statePart = [], action) => {
+const accessoriesReducer = (statePart = [], action: LoadAccessoriesAction) => {
```

Comment:
Technically, action can not only be of type `LoadAccessoriesAction` but rather of any action possible in our system.

Specify it as `ActionType`, imported from `store.ts` (_which creates circular dependancy, but lets not worry about that_)

Also you didn't specify the type of `statePart`. Afaik it is the only change needed to resolve the issue with `initialState`